src/tutorial/modal.component.tsx

```diff
@@ -39,20 +40,21 @@ const TutorialModal = ({ open, onClose }) => {
 
   return (
     <React.Fragment>
-      <ModalHeader closeModal={onClose} title={t('tutorial', 'Tutorial')}>
+      <ModalHeader className={styles.modalHeader} closeModal={onClose} title={t('tutorial', 'Tutorial')}>
         <p className={styles.description}>
           {t('modalDescription', 'Find walkthroughs and video tutorials on some of the core features of OpenMRS.')}
         </p>
       </ModalHeader>
-      <ModalBody className={styles.tutorialModal}>
+      <ModalBody className={styles.tutorialModal} hasScrollingContent >
```

Comment:
Can we remove this? by default wrapping up over `<React.Fragment>` make it scrollable.
You can read it here [#docs ](https://o3-docs.openmrs.org/docs/modal-system)
```suggestion
      <ModalBody className={styles.tutorialModal} >
```